src/pykiso/test_result/multi_result.py

```diff
@@ -0,0 +1,239 @@
+        """Call the stop function for all result classes."""
+        for result in self.result_classes:
+            result.stop()
+
+    def stopTest(self, test: Union[BasicTest, BaseTestSuite]) -> None:
+        """Call the stopTest function for all result classes.
+
+        :param test: running testcase
+        """
+        for result in self.result_classes:
+            result.stopTest(test)
+
+    def addSuccess(self, test: Union[BasicTest, BaseTestSuite]) -> None:
+        """Call the addSuccess function for all result classes.
+
+        :param test: running testcase
+        """
+        for result in self.result_classes:
+            result.addSuccess(test)
+
+    def addSkip(self, test: Union[BasicTest, BaseTestSuite], reason: str) -> None:
+        """Call the addSkip function for all result classes.
+
+        :param test: running testcase
+        """
+        for result in self.result_classes:
+            result.addSkip(test, reason)
+
+    def addUnexpectedSuccess(self, test: Union[BasicTest, BaseTestSuite]) -> None:
+        """Call the addUnexpectedSuccess function for all result classes.
+
+        :param test: running testcase
+        """
+        for result in self.result_classes:
+            result.addUnexpectedSuccess(test)
+
+    def addExpectedFailure(
+        self,
+        test: Union[BasicTest, BaseTestSuite],
+        err: ExcInfoType,
+    ) -> None:
+        """Call the addExpectedFailure function for all result classes.
+
+        :param test: running testcase
+        :param err: tuple returned by sys.exc_info
+        """
+        for result in self.result_classes:
+            result.addExpectedFailure(test, err)
+
+    def addSubTest(
+        self,
+        test: Union[BasicTest, BaseTestSuite],
+        subtest: _SubTest,
+        err: ExcInfoType,
+    ) -> None:
+        """Call the addSubTest function for all result classes.
+
+        :param test: running testcase
+        :param subtest: subtest runned
+        :param err: tuple returned by sys.exc_info
+        """
+        for result in self.result_classes:
+            result.addSubTest(test, subtest, err)
+
+    def addError(
+        self,
+        test: Union[BasicTest, BaseTestSuite],
+        err: ExcInfoType,
+    ) -> None:
+        """Call the addError function for all result classes.
+
+        :param test: running testcase
+        :param err: tuple returned by sys.exc_info
+        """
+        for result in self.result_classes:
+            result.addError(test, err)
+
+    def addFailure(
+        self,
+        test: Union[BasicTest, BaseTestSuite],
+        err: ExcInfoType,
+    ) -> None:
+        """Call the addFailure function for all result classes.
+
+        :param test: running testcase
+        :param err: tuple returned by sys.exc_info
+        """
+        for result in self.result_classes:
+            result.addFailure(test, err)
+
+    def printErrorList(self, flavour: str, errors: List[tuple]) -> None:
```

Comment:
KamelCase for unittest compatiility?

Reply:
Yes because the Class will be used for example as a resultclass in ``unittest.TextTestRunner``.